packages/examples/examples/notifications/snap.config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Use snap.config.js files instead of deprecated .json

Reply:
Ah, yes. Good point!